conf/parsers.conf

```diff
@@ -83,10 +83,10 @@
     Time_Key time
 
 [PARSER]
-    # http://rubular.com/r/izM6olvshn
-    Name crio
+    # http://rubular.com/r/tjUt3Awgg4
+    Name cri
     Format Regex
-    Regex /^(?<time>.+)\b(?<stream>stdout|stderr)\b(?<log>.*)$/
+    Regex ^(?<time>[^ ]+) (?<stream>stdout|stderr) (?<logtag>[^ ]*) (?<message>.*)$
     Time_Key    time
     Time_Format %Y-%m-%dT%H:%M:%S.%N%:z
     Time_Keep   On
```

Comment:
`Time_Keep On` seems to break Elasticsearch for me.
Fluent-Bit logs this error.
```
[2018/11/03 19:26:13] [ warn] [out_es] Elasticsearch error
{"took":150,"errors":true,"items":[{"index":{"_index":"logstash-2018.11.03","_type":"flb_type","_id":"_vEK22YBtlziBPpiPhwA","status":400,"error":{"type":"mapper_parsing_exception","reason":"failed to parse","caused_by":{"type":"i_o_exception","reason":"Duplicate field 'time'\n at [Source: org.elasticsearch.common.bytes.BytesReference$MarkSupportingStreamInputWrapper@2d5483; line: 1, column: 43]"}}}},{"index":{"_index":"logstash-2018.11.03","_type":"flb_type","_id":"__EK22YBtlziBPpiPhwA","status":400,"error":{"type":"mapper_parsing_exception","reason":"failed to parse","caused_by":{"type":"i_o_exception","reason":"Duplicate field 'time'\n at [Source: org.elasticsearch.common.bytes.BytesReference$MarkSupportingStreamInputWrapper@4988de6b; line: 1, column: 43]"}}}},{"index":{"_index":"logstash-2018.11.03","_type":"flb_type","_id":"APEK22YBtlziBPpiPh0A","status":400,"error":{"type":"mapper_parsing_exception","reason":"failed to parse","caused_b
```

Reply:
Doesn't make a difference for my use case, so I can happily remove that

Reply:
Thanks. Do you end up with additional time fields in Kafka?
I can also remove locally and wonder if one of the maintainers or someone with more experience with this field can advise. I haven't been able to grok the use case.